includes/discovery/os/barracuda.inc.php

```diff
@@ -17,4 +17,8 @@
     if (stristr($sysDescr, 'Barracuda Spam Firewall')) {
         $os = 'barracudaspamfirewall';
     }
+    if (stristr($sysDescr, 'Barracuda Firewall')) {
+        $os = 'barracudangfirewall';
```

Comment:
We have no $os definition for barracudangfirewall.

Full support will need to be added as per: http://docs.librenms.org/Support/FAQ/#faq8